Example/Tom/ViewController.swift

```diff
@@ -45,8 +47,9 @@ class ViewController: UIViewController {
         super.viewDidLoad()
 
         view.backgroundColor = retroCartoonsYellow
+        tomView.backgroundColor = retroCartoonsYellow
 
-        addTomView()
+//        addTomView()
```

Comment:
Could you please add a description something like that `remove comment to programmatically implementation work`